docs/source/drivers-clients/http-client-server-api.rst

```diff
@@ -66,20 +96,129 @@ The HTTP API currently exposes two endpoints, one to get information about a spe
+                    "fid":0,
+                    "fulfillment":"cf:4:__Y_Um6H73iwPe6ejWXEw930SQhqVGjtAHTXilPp0P01vE_Cx6zs3GJVoO1jhPL18C94PIVkLTGMUB2aKC9qsbIb3w8ejpOf0_I3OCuTbPdkd6r2lKMeVftMyMxkeWoM",
+                    "input":{
+                        "cid":0,
+                        "txid":"598ce4e9a29837a1c6fc337ee4a41b61c20ad25d01646754c825b1116abd8761"
+                    }
+                }
+            ],
+            "operation":"TRANSFER",
+            "timestamp":"1471423869",
+            "version":1
+         }
+      }
 
    **Example response**:
 
    .. sourcecode:: http
 
       HTTP/1.1 201 Created
       Content-Type: application/json
-      TODO: Other headers?
 
-      (TODO) Insert example response body here
+      {
+        "assignee":"4XYfCbabAWVUCbjTmRTFEu2sc3dFEdkse4r6X498B1s8",
+        "id":"7ad5a4b83bc8c70c4fd7420ff3c60693ab8e6d0e3124378ca69ed5acd2578792",
+        "transaction":{
+            "conditions":[
+                {
+                    "cid":0,
+                    "condition":{
+                        "details":{
+                            "bitmask":32,
+                            "public_key":"CwA8s2QYQBfNz4WvjEwmJi83zYr7JhxRhidx6uZ5KBVd",
+                            "signature":null,
+                            "type":"fulfillment",
+                            "type_id":4
+                        },
+                        "uri":"cc:4:20:sVA_3p8gvl8yRFNTomqm6MaavKewka6dGYcFAuPrRXQ:96"
+                    },
+                    "new_owners":[
+                        "CwA8s2QYQBfNz4WvjEwmJi83zYr7JhxRhidx6uZ5KBVd"
+                    ]
+                }
+            ],
+            "data":{
+                "payload":null,
+                "uuid":"a9999d69-6cde-4b80-819d-ed57f6abe257"
+            },
+            "fulfillments":[
+                {
+                    "current_owners":[
+                        "JEAkEJqLbbgDRAtMm8YAjGp759Aq2qTn9eaEHUj2XePE"
+                    ],
+                    "fid":0,
+                    "fulfillment":"cf:4:__Y_Um6H73iwPe6ejWXEw930SQhqVGjtAHTXilPp0P01vE_Cx6zs3GJVoO1jhPL18C94PIVkLTGMUB2aKC9qsbIb3w8ejpOf0_I3OCuTbPdkd6r2lKMeVftMyMxkeWoM",
+                    "input":{
+                        "cid":0,
+                        "txid":"598ce4e9a29837a1c6fc337ee4a41b61c20ad25d01646754c825b1116abd8761"
+                    }
+                }
+            ],
+            "operation":"TRANSFER",
+            "timestamp":"1471423869",
+            "version":1
+        }
+      }
 
    :statuscode 201: A new transaction was created.
+   :statuscode 400: The transaction was invalid and not created.
+
+   **Disclaimer**
+
+   ``CREATE`` transactions are treated differently from ``TRANSFER`` assets.
+   The reason is that a ``CREATE`` transaction needs to be signed by the federation nodes and not by the client.
+
+   The following python snippet at the client can be used to generate ``CREATE`` transactions before they can be pushed to the API server:
```

Comment:
> For client code examples, it may be preferable to avoid including/requiring server code.

yeah, but this is the server ofc, and there is a special case on how to push transactions...
you mean dropping this entire disclaimer?